cmake/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ option(tensorflow_C_PACKAGE_PATH "Path to tensorflow C package installation dir"
 option(onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS "Enable operator implemented in language other than cpp" OFF)
 option(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS "Dump node input shapes and output data to standard output when executing the model." OFF)
 option(onnxruntime_USE_DML "Build with DirectML support" OFF)
+option(onnxruntime_USE_WINML "Build with WinML support" OFF)
```

Comment:
>onnxruntime_USE_WINML [](start = 7, length = 21)

this is new?

Reply:
Already existed, but wasn't declared :)

Reply:
Like it was being accessed in the CMake scripts but wasn't documented as an option